libs/utils/utils.js

```diff
@@ -689,6 +690,31 @@ export function convertStageLinks({ anchors, config, hostname, href }) {
   });
 }
 
+async function getImsCountry() {
+  if (window.adobeIMS?.isSignedInUser()) {
+    const profile = await window.adobeIMS.getProfile();
+    return profile.countryCode;
+  }
+
+  return null;
+}
+
+export async function generateM7Link() {
```

Comment:
This method should not be in `utils`.

Reply:
I moved all my changes to autoblock `blocks/m7business/m7business.js`